### PR TITLE
adding rules based on more PMA labs

### PR DESCRIPTION
**Pull request by mr-tz**
addresses #296

### CODE REVIEW

File: host-interaction/mutex/check-mutex-and-exit.yml [hunk 1]
```diff
@@ -11,15 +11,20 @@ rule:
...
-        - and:
-          - api: GetLastError
-          - or:
-            - number: 2 = ERROR_FILE_NOT_FOUND
-            - number: 0xB7 = ERROR_ALREADY_EXISTS
-            - number: 5 = ERROR_ACCESS_DENIED
+        - api: ExitProcess
+        - api: exit
+        - api: _Exit
+        - api: _exit
```
Comment (mr-tz) [inline host-interaction/mutex/check-mutex-and-exit.yml:22]: leverages function ID!

File: load-code/pe/parse-pe-header.yml [hunk 2]
```diff
@@ -32,3 +32,16 @@ rule:
...
+          - 3 or more:
+            - offset: 0x4 = IMAGE_NT_HEADERS.FileHeader.Machine
+            - offset: 0x6 = IMAGE_NT_HEADERS.FileHeader.NumberOfSections
+            - offset: 0x14 = IMAGE_NT_HEADERS.FileHeader.SizeOfOptionalHeader
+            - offset: 0x16 = IMAGE_NT_HEADERS.FileHeader.Characteristics
+            - offset: 0x28 = IMAGE_NT_HEADERS.OptionalHeader.AddressOfEntryPoint  # for 32 and 64 bit
+            - offset/x32: 0x34 = IMAGE_NT_HEADERS.OptionalHeader.ImageBase
+            - offset/x32: 0x50 = IMAGE_NT_HEADERS.OptionalHeader.SizeOfImage
+            - offset/x64: 0x30 = IMAGE_NT_HEADERS.OptionalHeader.ImageBase
+            - offset/x64: 0x50 = IMAGE_NT_HEADERS64.OptionalHeader.SizeOfImage
```
Comment (mr-tz) [inline load-code/pe/parse-pe-header.yml:47]: afraid of FPs here, may switch to
```
- 3 or more:
  - basic block:
    - and:
      - 0x3C
    - or:
      - offsets...
```

**Comment by mr-tz** [comment]
waiting with this one till after the release, reviews are welcome though

File: anti-analysis/anti-forensic/self-deletion/self-delete.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 rule:
   meta:
-    name: self delete via COMSPEC environment variable
+    name: self delete
```
Comment (mr-tz) [inline anti-analysis/anti-forensic/self-deletion/self-delete.yml:3]: maybe `self delete via cmd`?

File: host-interaction/process/create/execute-command.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    mbc:
+      - Process::Create Process [C0017]
+    examples:
+      - Practical Malware Analysis Lab 11-03.exe_:0x4012D0
+  features:
+    - or:
+      - api: system
+      - api: _system
+      - api: wsystem
+      - api: _wsystem
```
Comment (mr-tz) [inline host-interaction/process/create/execute-command.yml:16]: possible thanks to function ID

**Review by williballenthin — APPROVED** [review]

**Comment by williballenthin** [comment]
viv doesn't recognize `_system`:
image

its a statically linked routine that would have to be matched by FLIRT:
image

**Comment by mr-tz** [comment]
thanks!!

**Pull request merged**